plugins/janus_videoroom.c

```diff
@@ -6882,12 +6882,18 @@ static void *janus_videoroom_handler(void *data) {
 				janus_refcount_decrease(&subscriber->ref);
 				goto error;
 			} else if(!strcasecmp(request_text, "start")) {
+				janus_videoroom_publisher *feed = subscriber->feed;
+
 				/* Start/restart receiving the publisher streams */
 				if(subscriber->paused && msg->jsep == NULL) {
 					/* This is just resuming a paused stream, reset the RTP sequence numbers */
 					subscriber->context.a_seq_reset = TRUE;
 					subscriber->context.v_seq_reset = TRUE;
 				}
+				if(feed && g_atomic_int_get(&feed->started, 1)) {
+						/* Send a FIR */
+						janus_videoroom_reqpli(feed, "Subscriber start");
+				}
```

Comment:
This should be in the `if(subscriber->paused` above, though. Indentation should be fixed too.
As a side note, I'd move the `*feed` initialization right before the check, since it's unused if we're not in `subscriber->paused`.

Reply:
ok, moved inside the paused if, along with the needed initialization and indentation

Reply:
something is wrong into the `g_atomic_int_get(&feed->started, 1)`, investigating on it

Reply:
> something is wrong into the `g_atomic_int_get(&feed->started, 1)`, investigating on it

ok, it was `if(feed && feed->session && g_atomic_int_get(&feed->session->started)) {`

Reply:
fixed